src/views/Form/index.tsx

```diff
@@ -40,40 +113,108 @@ const Form = () => {
+    if (event.key === 'Enter') {
+      handleSubmit();
+    }
   };
 
   const navigate = useNavigate();
   const goBack = () => {
     navigate('/');
   };
 
+  const deleteByIndex = (idx: number): void => {
+    const userToDelete = users[idx];
+    deleteUser(userToDelete.firstName, userToDelete.lastName);
+  };
+
+  React.useEffect(() => {
+    fetchData();
+  }, []);
+
   return (
     <Grid container spacing={4}>
       <Grid item xs={12}>
         <h1>NR Front End Form</h1>
         <p>
-          This is a simple test form, please write your information correctly.
+          This is a very simple form. Please, fill your first and last name then hit Submit!
         </p>
       </Grid>
       {
-        submitted && firstIsValid && lastIsValid ? <Grid item xs={12}><Alert type="success" styling="bcgov-success-background" element="Your name was submitted to nowhere!" /></Grid> : ''
+        showError && (
+        <Grid item xs={12}>
+          <Alert icon={<MdCancel size={32} />} type="error" styling="bcgov-error-background" element={errorMessage} />
+        </Grid>
+        )
       }
       {
-        submitted && !(firstIsValid && lastIsValid) ? <Grid item xs={12}><Alert type="error" styling="bcgov-error-background" element="Please, enter your name correctly!" /></Grid> : ''
+        loading && (
+          <Grid item xs={12}>
+            <CardLoader />
+          </Grid>
+        )
       }
       <Grid item xs={12} sm={6}>
-        <Label labelStr="First Name" forStr="fnInput" />
-        <TextInput id="fnInput" isValid={firstIsValid} feedback={firstFeed} blur={handleFirstName} />
+        <Label labelStr="First name" forStr="first-name" />
+        <TextInput
+          id="first-name"
+          isValid={firstIsValid}
+          feedback={firstFeed}
+          onChangeHandler={handleFirstName}
+          inputValue={firstName}
+          onKeyDownHandler={enterKeyHandler}
+          disabled={loading}
+        />
       </Grid>
       <Grid item xs={12} sm={6}>
-        <Label labelStr="Last Name" forStr="lnInput" />
-        <TextInput id="lnInput" isValid={lastIsValid} feedback={lastFeed} blur={handleLastName} />
+        <Label labelStr="Last lame" forStr="last-name" />
+        <TextInput
+          id="last-name"
+          isValid={lastIsValid}
+          feedback={lastFeed}
+          onChangeHandler={handleLastName}
+          inputValue={lastName}
+          onKeyDownHandler={enterKeyHandler}
+          disabled={loading}
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <Button
+          onClick={handleSubmit}
+          label="Submit"
+          styling="bcgov-normal-blue btn buttonMargin"
+          disabled={loading}
+        />
+        <Button
+          onClick={resetForm}
+          label="Reset"
+          styling="bcgov-normal-white btn buttonMargin"
+          disabled={loading}
+        />
+        <Button onClick={goBack} label="Back to home" styling="bcgov-normal-white btn buttonMargin" />
```

Comment:
I think this button should follow the same pattern as the other in the page (each attribute on its own line).

Reply:
Thanks for noticing!